src/Error.php

```diff
@@ -59,10 +59,10 @@ public function __construct($message = null, $code = self::ERROR_OTHER, $data =
      *
      * If the error code is 0, it will be set to -32000 (ERROR_OTHER).
      *
-     * @param  int $code
+     * @param mixed $code
      * @return self
      */
-    public function setCode($code)
+    public function setCode($code): Error
```

Comment:
add return type is bc break as class is not final, see https://3v4l.org/d8L2h

Reply:
ok, I'm removing it 👍